### DISCUSSION TITLE
Box explosion in NPT simulation

### DISCUSSION THREAD

**Discussion by yangqifanshihaoren** [mir-group/nequip#421] [Q&A]
When I use the trained model in lammps for npt simulation to calculate shear stress-strain curve, I found the box always explode to an extremely large volume(about 1000 times the previous amount). The training dataset is obtained by relaxing progress in vasp with "PSTRESS = 10" and "PSTRESS = -10" to get structures of different volumes. Is that because the training dataset is too little? Or because when using "erate" in lammps, the nequip trained model can't perform well? Thank you!
The following picture is the input file of the npt simulation.
![1713669175790](https://github.com/mir-group/nequip/assets/167204963/acc8e594-bcca-4acb-8d5e-86ca3aa2a6aa)

**Comment by yangqifanshihaoren** [comment]
I didn't use stress as a loss function but only energy and force since I haven't figure out the unit of stress I should use clearly. But I used the "StressForceOutput" so the stress is derived from energy and force. 
It is confused which part resulted the "box explosion" problem. I don't think it is because the original structure of npt simulation is unreasonable because I relaxed it first using nequip trained model in lammps.

**Reply by yangqifanshihaoren** [reply]
Does this is because I didn't use stress as the loss function?

**Comment by yangqifanshihaoren** [comment]
@Linux-cpp-lisp Could you please answer my quetion? Thank you very much!
